gcloud/pubsub/_gax.py

```diff
@@ -46,13 +47,21 @@ def list_topics(self, project):
         :type project: string
         :param project: project ID
 
+        :type page_token: string
+        :param page_token: opaque marker for the next "page" of topics. If not
+                           passed, the API will return the first page of
+                           topics.
+
         :rtype: tuple, (list, str)
         :returns: list of ``Topic`` resource dicts, plus a
                   "next page token" string:  if not None, indicates that
                   more topics can be retrieved with another call (pass that
                   value as ``page_token``).
         """
-        options = CallOptions(is_page_streaming=False)
+        if page_token is None:
```

Comment:
Do you think this should/could moved to a `build_options()` method since setting `'page_token'` happens a few places it might help for debugging/future work if there was only one place to update that?